### DISCUSSION TITLE
Antenna gain

### DISCUSSION THREAD

**Discussion by amira123bd** [NVlabs/sionna#454] [Q&A]
Hello everyone, 
I want to ask if there is a way to access the antenna gain in sionna ?

**Comment by faycalaa** [comment]
Hi,
The [Antenna class](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Antenna) features utilities to compute and visualise antenna patterns.

**Reply by amira123bd** [reply]
Thanks, and is there a way to act on the gain values ?

**Reply by faycalaa** [reply]
When creating an `Antenna` instance, one can specify a custom antenna pattern by setting the `pattern` parameter to a callable. This callable should take as input vectors of zenith and azimuth angles of the same length and return the corresponding zenith and azimuth patterns for each pair.
